crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1138,18 +1138,18 @@ impl ScheduleGraph {
                     ambiguous_with_flattened.add_edge(lhs, rhs, ());
                 }
                 (NodeId::Set(_), NodeId::System(_)) => {
-                    for &lhs_ in set_systems.get(&lhs).unwrap() {
+                    for &lhs_ in set_systems.get(&lhs).unwrap_or(&vec![]) {
```

Comment:
It's unclear if this allocates or not. Could we use `&Vec::new()` instead?

Reply:
FWIW, `vec![]` with no parameters expands to `Vec::new()` and therefore doesn't allocate (see https://doc.rust-lang.org/src/alloc/macros.rs.html#42) (but by all means change it if you think it's clearer).

Reply:
```suggestion
                    for &lhs_ in set_systems.get(&lhs).unwrap_or(&Vec::new()) {
```

I think this is slightly clearer but I super don't care :p

Reply:
🤷